sdk/core/azure-core/test/ut/transport_adapter.hpp

```diff
@@ -1,6 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
+/**
```

Comment:
Consider naming this something specific to tests to make clear this is a test only class

Reply:
we would then need to rename all the test files, to be consistent.